src/FiltrationGroupComponent/FiltrationGroupComponent.tsx

```diff
@@ -39,9 +32,6 @@ const FiltrationGroupComponent: React.FC<Props & WithOutsideClickProps> = ({
 	addSimpleConditionButton,
 	onConditionStateToggle
 }) => {
-	const [horizontalBracket, setHorizontalBracket] = useState<
-		ItemsRootElement[]
-	>([]);
 	const [verticalBracket, setVerticalBracket] = useState<boolean>(false);
 	const context = useContext(FilterWrapperContext);
 	const shouldRerenderBrackets = React.useRef(false);
```

Comment:
Наверху уже полно всего достаём из React, это тоже надо тогда
```suggestion
	const shouldRerenderBrackets = useRef(false);
```